src/sinopiaServer.js

```diff
@@ -193,5 +193,11 @@ export const getSearchResults = async (query, queryFrom = 0) => {
         title: row._source.label,
       })),
     }))
-    .catch(error => console.error(error))
+    .catch(error => {
+      return ({
+        totalHits: 0,
+        results: [],
+        error: 'There was a problem retrieving search results. Please try again later.',
```

Comment:
Please provide the actual error. And remove `Please try again later` as it may not work later depending on the error.

Reply:
> Also, should display message similar to other warnings. (Top of screen, dismissable, full width.)
> 
top of screen is fine, but If the error message is dismissed, and then the user tries subsequent search (without refreshing the page) the error will not reappear. This drawback can maybe overcome by changing SearchResultsMessage() to a Component with constructor props (I tried and failed to make it work using react state hooks), but I ultimately left out the dismiss-able element because (as seen in the GIF) if you perform a successful search (or even just a successful connection to search with zero results) the message clears itself. That said, I am fine trying to re-work the component if making it dismiss-able is a priority.

Reply:
Dismissable is not a top priority.

Reply:
I found a way to do it using react hooks and drawing from the example in the Editor component.